src/vellum/workflows/inputs/base.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, **kwargs: Any) -> None:
                 origin = get_origin(field_type)
                 args = get_args(field_type)
                 if not (origin is Union and type(None) in args):
-                    raise WorkflowInitializationException(
+                    raise NodeException(
```

Comment:
[blocking] This is not an exception raised by a node, but rather by the Workflow, so we don't want to raise a Node exception. Rather, we want the consumer of the sdk to not trigger a sentry alert when we detect workflow initialization exceptions